### PR TITLE
Add option to deploy plugin to WP.org using GitHub Actions

### PR DESCRIPTION
**Pull request by sudhir-yadav**
Add option to deploy plugin to WP.org using GitHub Actions

### CODE REVIEW

File: .github/main.workflow [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    args = "tag"
+}
+
+action "WordPress Plugin Deploy" {
+  needs = ["tag"]
+  uses = "rtCamp/action-wp-org-plugin-deploy@master"
+  secrets = ["WORDPRESS_USERNAME", "WORDPRESS_PASSWORD"]
+  env = {
+    SLUG = "transcoder"
+    ASSETS_DIR = "assets"
```
Comment (thrijith) [inline .github/main.workflow]: `assets` directory is not available in the repo, so we can remove this.

File: .github/main.workflow [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    EXCLUDE_LIST = "deploy.sh deploy-common.sh readme.sh README.md .gitattributes .gitignore map.conf nginx.log tests bin assets CONTRIBUTING.md Gruntfile.js bootstrap_tests.php config.rb log package.json phpunit.xml phpunit.xml.dist phpunit.xml"
```
Comment (thrijith) [inline .github/main.workflow]: Add `node_modules`, `.jsrc`, `.travis.yml` to this list too.

File: .github/main.workflow [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    args = "tag"
+}
+
+action "WordPress Plugin Deploy" {
+  needs = ["tag"]
+  uses = "rtCamp/action-wp-org-plugin-deploy@master"
+  secrets = ["WORDPRESS_USERNAME", "WORDPRESS_PASSWORD"]
+  env = {
+    SLUG = "transcoder"
+    EXCLUDE_LIST = "deploy.sh deploy-common.sh readme.sh README.md .gitattributes .gitignore map.conf nginx.log tests bin assets CONTRIBUTING.md Gruntfile.js bootstrap_tests.php config.rb log package.json phpunit.xml phpunit.xml.dist phpunit.xml node_modules .jsrc .travis.yml"
```
Comment (thrijith) [inline .github/main.workflow]:
```suggestion
    EXCLUDE_LIST = "deploy.sh deploy-common.sh readme.sh README.md .gitattributes .gitignore map.conf nginx.log tests bin assets CONTRIBUTING.md Gruntfile.js bootstrap_tests.php config.rb log package.json phpunit.xml phpunit.xml.dist phpunit.xml node_modules .jscsrc .travis.yml"
```

**Comment by SID177** [comment]
Created new PR https://github.com/rtCamp/transcoder/pull/197 so closing this one.

**Pull request closed without merging**